Sources/Network/URLSessionNetworkStack.swift

```diff
@@ -145,17 +145,19 @@ public extension Network {
 
                 switch (httpStatusCode, data) {
                 case (.success, let remoteData?):
-                    completion(.success(remoteData))
+                    completion(.success(Network.Value(value: remoteData, response: httpResponse)))
```

Comment:
we should propagate `response` and not `httpResponse` (even though it is the same instance)

(more cases below)